app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -526,7 +538,39 @@ export default class Heatmap {
     d3.event.stopPropagation();
   }
 
-  pan(deltaX, deltaY) {
+  scrollToFocus() {
```

Comment:
If you accept my suggestion above this would be called `scrollToRow` or something.

Reply:
Done!